homeassistant/components/verisure/coordinator.py

```diff
@@ -56,60 +62,111 @@ async def async_login(self) -> bool:
 
     async def async_logout(self, _event: Event) -> None:
         """Logout from Verisure."""
-        try:
-            await self.hass.async_add_executor_job(self.verisure.logout)
-        except VerisureError as ex:
-            LOGGER.error("Could not log out from verisure, %s", ex)
 
     async def _async_update_data(self) -> dict:
         """Fetch data from Verisure."""
         try:
             overview = await self.hass.async_add_executor_job(
-                self.verisure.get_overview
+                self.verisure.request,
+                self.verisure.arm_state(),
+                self.verisure.broadband(),
+                self.verisure.cameras(),
+                self.verisure.climate(),
+                self.verisure.door_window(),
+                self.verisure.smart_lock(),
+                self.verisure.smartplugs(),
             )
-        except VerisureResponseError as ex:
-            LOGGER.error("Could not read overview, %s", ex)
-            if ex.status_code == HTTPStatus.SERVICE_UNAVAILABLE:
-                LOGGER.info("Trying to log in again")
-                await self.async_login()
-                return {}
-            raise
+        except VerisureResponseError as err:
+            LOGGER.debug("Cookie expired or service unavailable, %s", err)
+            overview = self._overview
+            try:
+                await self.hass.async_add_executor_job(self.verisure.update_cookie)
+            except VerisureResponseError as ex:
+                LOGGER.error("Credentials for Verisure expired, %s", ex)
```

Comment:
Info (or debug) log since this is expected to happen (yearly)?

Reply:
Good comment, as it will raise and trigger the UI, I don't think logging might be needed at all.

Reply:
I'll remove the logging then